content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,134 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the Git user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+3. Don't forget to install DVC and any other requirements in your environment!
+
+   ```bash
+   pip install dvc dvclive
+   ```
+
+## Notebooks
+
+After completing the setup, you can work with DVC in SageMaker notebooks like
+you would in any other environment. Take a look at DVC [experiments] for how to
+get started with DVC in notebooks.
+
+If you would like to see live experiment updates in [DVC Studio], get your
+[token] and save it in your [dvc config] or `DVC_STUDIO_TOKEN` environment
+variable. For example, to set it globally for all of a user's projects:
+
+```cli
+$ dvc config --global studio.token ***
+```
+
+While the experiment runs, you will see live updates like this in Studio:
```

Comment:
A minor thing but "in Studio" could be confusing for people who use Sagemaker Studio. Even in your gif you actually switch between Sagemaker Studio and Iterative Studio.

Reply:
I was trying to specify DVC Studio everywhere, but looks like I missed this one, thanks!